### PR TITLE
added pros & cons

### PR DESCRIPTION
**Pull request by Praneeth16**
Added advantages and disadvantages for Random Forests.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/machine-learning/random-forest/index.md [hunk 1]
```diff
@@ -49,6 +49,18 @@ predictions2 = model2.predict(data_test)
 print(predictions2)
 ```
 
+### Advantages of Random Forests
+
+- It generally does not overfit too badly, and it is very easy to stop it from overfitting.
```
Comment (jonathan-grah) [inline guide/english/machine-learning/random-forest/index.md]:
```
- They generally don't overfit too badly, and it's very easy to stop.
```

File: guide/english/machine-learning/random-forest/index.md [hunk 2]
```diff
@@ -49,6 +49,18 @@ predictions2 = model2.predict(data_test)
...
+- It has few, if any, `statistical assumptions`. It does not assume that your data is normally distributed, the relationship is linear, or you have specified interactions.
+- It requires very few pieces of feature engineering.
+
+### Disadvantages of Random Forests
+
+- It doesnot work well on image data.
```
Comment (jonathan-grah) [inline guide/english/machine-learning/random-forest/index.md]:
```
- Random forests don't work well on image data.
```

File: guide/english/machine-learning/random-forest/index.md [hunk 3]
```diff
@@ -49,6 +49,18 @@ predictions2 = model2.predict(data_test)
...
+- It is a predictive modeling tool and not a descriptive tool. That means, if you are looking for a description of the relationships in your data, other approaches would be preferred.
```
Comment (jonathan-grah) [inline guide/english/machine-learning/random-forest/index.md]:
```
- Predictive modelling tool, not a descriptive tool, meaning if you are looking for a description of the relationships in your data; other approaches would be preferred.
```

File: guide/english/machine-learning/random-forest/index.md [hunk 4]
```diff
@@ -49,6 +49,18 @@ predictions2 = model2.predict(data_test)
...
+
+
```
Comment (jonathan-grah) [inline guide/english/machine-learning/random-forest/index.md]: Remove this line.

File: guide/english/machine-learning/random-forest/index.md [hunk 5]
```diff
@@ -49,6 +49,18 @@ predictions2 = model2.predict(data_test)
 print(predictions2)
 ```
 
+### Advantages of Random Forests
+
+- It generally does not overfit too badly, and it is very easy to stop it from overfitting.
+- It has few, if any, `statistical assumptions`. It does not assume that your data is normally distributed, the relationship is linear, or you have specified interactions.
+- It requires very few pieces of feature engineering.
```
Comment (jonathan-grah) [inline guide/english/machine-learning/random-forest/index.md]:
```
- Requires very few pieces of feature engineering.
```

**Review by jonathan-grah — CHANGES_REQUESTED** [review]
### LGTM 👍 
Just a few minor SPG changes 😃

**Comment by RandellDawson** [comment]
@jonathan-grah This PR has gone stale.  Do you want to make your suggested changes on behalf of the user and merge or just close it?

**Comment by RandellDawson** [comment]
HI @Praneeth16 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**